src/protagonist/API/Features/Storage/Requests/GetImageStorage.cs

```diff
@@ -0,0 +1,46 @@
+﻿using API.Infrastructure.Requests;
+using DLCS.Core.Types;
+using DLCS.Model.Assets;
+using DLCS.Repository;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace API.Features.Storage.Requests;
+
+public class GetImageStorage : IRequest<FetchEntityResult<ImageStorage>>
+{
+    public int CustomerId { get; }
+    
+    public int SpaceId { get; }
+    
+    public string ImageId { get; }
+    
+    public GetImageStorage(int customerId, int spaceId, string imageId)
+    {
+        CustomerId = customerId;
+        SpaceId = spaceId;
+        ImageId = imageId;
+    }
+}
+
+public class GetImageStorageHandler : IRequestHandler<GetImageStorage, FetchEntityResult<ImageStorage>>
+{
+    private readonly DlcsContext dbContext;
+
+    public GetImageStorageHandler(DlcsContext dbContext)
+    {
+        this.dbContext = dbContext;
+    }
+
+    public async Task<FetchEntityResult<ImageStorage>> Handle(GetImageStorage request, CancellationToken cancellationToken)
+    {
+        var assetId = AssetId.FromString($"{request.CustomerId}/{request.SpaceId}/{request.ImageId}");
```

Comment:
Use the constructor here (`new AssetId(request.CustomerId, request.SpaceId, request.ImageId);`) rather than factory method. The intention of that `.FromString()` is to help when we only have the string, e.g. when we read from DB. Here we have the component parts already.

Creating a string to create an `AssetId` feels unnecessary when we can just call the constructor. Internally `FromString` will call the same ctor after parsing the string anyway.